servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilder.java

```diff
@@ -63,6 +64,9 @@
  * A builder of {@link StreamingHttpClient} instances which have a capacity to call any server based on the parsed
  * absolute-form URL address information from each {@link StreamingHttpRequest}.
  * <p>
+ * If {@link HttpRequestMetaData#requestTarget()} is not an absolute-form URL, a {@link MalformedURLException} will be
```

Comment:
Possibly just a personal preference but I usually prefer `URISyntaxException` to `MalformedURLException` because the later often gets handled as a generic `IOException` by lazy people who hate checked exceptions.

Reply:
I've opened a separate PR #1955 for this change to make it move visible in release notes